config/settings.yml

```diff
@@ -911,6 +911,7 @@
   :maindb: ExtManagementSystem
   :container_deployment_wizard: false
 :prototype:
+  :queue_type: artemis
```

Comment:
Should this really be committed as the default queue type? I figured this would be opt-in.

(Unless of course, this will be removed before this leaves WIP)

Reply:
miq_queue should be the default.